.dockerignore

```diff
@@ -0,0 +1,4 @@
+Dockerfile
+README.md
+node_modules
```

Comment:
Because our build process excludes everything in `.gitignore` at the moment. But maybe you are right. In any event the file will be deleted soon. Fuck it.